numerical/area/bounds/cartesian.py

```diff
@@ -0,0 +1,86 @@
+from numerical.area.bounds import _base
+import matplotlib.pyplot as plt
+
+
+class CartesianCoordinates:
+    COORDS_TYPE = "cartesian"
+
+
+class LineBoundary1D(_base.BoundaryIterable, CartesianCoordinates):
+    def __init__(self, x_start, x_end):
+        super().__init__(
+            [
+                _base.LineBoundary(x_start, x_end),
+            ]
+        )
+        self._set_described_rect()
+
+    def _set_described_rect(self):
```

Comment:
What is `described_rect` for circle?

Reply:
Since it is a polar coordinate then described rectangular coincide with a defined boundary. For CircleBoundary and HalfCircleBoundary, this method is inherited from CircleSegmentBoundary.